packages/eslint-plugin/tests/rules/no-useless-template-literals.test.ts

```diff
@@ -0,0 +1,194 @@
+import { RuleTester } from '@typescript-eslint/rule-tester';
+
+import rule from '../../src/rules/no-useless-template-literals';
+import { getFixturesRootDir } from '../RuleTester';
+
+const rootPath = getFixturesRootDir();
+
+const ruleTester = new RuleTester({
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    tsconfigRootDir: rootPath,
+    project: './tsconfig.json',
+  },
+});
+
+ruleTester.run('no-useless-template-literals', rule, {
+  valid: [
+    "const string = 'a';",
+
+    // allow variables & literals concatenation
```

Comment:
[Style] Not a blocker IMO, but a preference - I don't think there's a need for the comments in the test file explaining sections of tests. They tests are pretty self-explanatory. And over time these comments have a bad habit of falling out of sync.

```suggestion
```

Reply:
yup, you're probably right
I just can't get used to seeing this type of tests instead of seeing something like `it('should work...')` 😂 
anyway, removed (I think losing sync is a good point here)